src/views/Header.js

```diff
@@ -306,30 +326,26 @@ const styles = StyleSheet.create({
     elevation: 4,
   },
   appBar: {
-    flex: 1
+    flex: 1,
+    position: 'relative',
   },
-  item: {
+  header: {
     flexDirection: 'row',
-    alignItems: 'center',
   },
-  title: {
-    bottom: 0,
-    left: 40,
-    position: 'absolute',
-    right: 40,
-    top: 0,
+  item: {
+    justifyContent: 'center',
```

Comment:
Since everything is `column` by default, justifyContent will center items vertically. That's the desired effect. We don't want title to stick to top.